### PR TITLE
[ZEPPELIN-2770] Zeppelin is not working in IE-11

### PR DESCRIPTION
**Pull request by byamthev**
### What is this PR for?
There is a javascript error while loading zeppelin homepage in IE-11, causing a blank page to appear. 

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
* [ZEPPELIN-2770](https://issues.apache.org/jira/browse/ZEPPELIN-2770)

### How should this be tested?
Try opening Zeppelin homepage in IE-11.

### Questions:
* Does the licenses files need update? N/A
* Is there breaking changes for older versions? N/A
* Does this needs documentation? N/A

### CODE REVIEW

File: zeppelin-web/src/app/app.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ function auth () {
       $rootScope.ticket = angular.fromJson(response.data).body
 
       $rootScope.ticket.screenUsername = $rootScope.ticket.principal
-      if ($rootScope.ticket.principal.startsWith('#Pac4j')) {
+      if ($rootScope.ticket.principal.toString().indexOf('#Pac4j') === 0) {
```
Comment (1ambda) [inline zeppelin-web/src/app/app.js]: Do we need `toString`? except for LGTM.

**Comment by 1ambda** [comment]
Thanks for the contribution @byamthev I left a comment. Could you check that?

File: zeppelin-web/src/app/app.js (hunk 1 above)
Comment (byamthev) [inline zeppelin-web/src/app/app.js]: You are right, we don't need toString, I will remove it.

**Comment by byamthev** [comment]
@1ambda I checked your comment and corrected, thanks. (note: I restarted the CI job that failed because of flaky tests)

**Comment by 1ambda** [comment]
LGTM. merge if no more discussion.

**Pull request closed without merging**